lib/pubid/iso/parser.rb

```diff
@@ -6,13 +6,16 @@ class Parser < Pubid::Core::Parser
     STAGES = %w[NP NWIP WD CD PRF AWI PWI FPD].freeze
     TYPES = %w[DATA ISP IWA R TTA TS TR IS PAS Guide GUIDE DIR].freeze
     # TYPED_STAGES = %w[DIS FDIS DPAS FDTR FDTS DTS DTR PDTR PDTS].freeze
-    SUPPLEMENTS = %w[Amd Cor AMD COR].freeze
+    SUPPLEMENTS = %w[Amd Cor AMD COR Suppl].freeze
```

Comment:
In `lib/pubid/iso/identifier/supplement.rb` there is a line `values: %w[Suppl SUP]`. But in this line, there is only `Suppl` yet there is no `SUP`. Does this mean pubids with `SUP` (as seen when using the Directives base type) cannot be parsed like before? Can we support both?

Reply:
> In `lib/pubid/iso/identifier/supplement.rb` there is a line `values: %w[Suppl SUP]`. But in this line, there is only `Suppl` yet there is no `SUP`. Does this mean pubids with `SUP` (as seen when using the Directives base type) cannot be parsed like before? Can we support both?

Dir parser is not using `SUPPLEMENTS`, I added another constant `DIR_SUPPLEMENTS` so it's more clear now.

Reply:
Got it, thank you!